sources/PTYTextView.h

```diff
@@ -331,6 +331,10 @@ typedef NS_ENUM(NSInteger, PTYTextViewSelectionExtensionUnit) {
 typedef void (^PTYTextViewDrawingHookBlock)(iTermTextDrawingHelper *);
 @property(nonatomic, copy) PTYTextViewDrawingHookBlock drawingHook;
 
+@property(nonatomic, assign) BOOL showTimestamps;
```

Comment:
Why was it necessary to expose these properties?

Reply:
I'm using these in my embeddable iTerm2 control to change the title/state of menu items.
For the `showTimestamps` case I'm showing `Hide Timestamps` if it's `YES` and `Show Timestamps` if it's `NO`.